README.md

```diff
@@ -12,6 +12,13 @@ This repository contains a test suite for CapTP, OCapN netlayers & OCapN URIs, a
 - Tor
 - python-stem (https://stem.torproject.org/)
 
+### TCP netlayer
```

Comment:
We should rename this to something like "TCP Testing netlayer (Do not use in production)!"

Reply:
Sure, «testing netlayer» in «ocapn-test-suite» it is)